### PR TITLE
Engineering Roundstart Changes 2.0

### PR DESCRIPTION
**Pull request by furrycactus**
(Remade this PR since there were some issues I couldn't work out and fix with the first one.)

This PR was made after I did some discussing with Kyres1 about his suggestion thread, which can be found here: 
- https://forums.aurorastation.org/viewtopic.php?f=18&t=12095

Also added this suggestion to the PR as well:
- https://forums.aurorastation.org/viewtopic.php?f=18&t=12129

It effectively eliminates a lot of unnecessary tedium that engineers have to deal with every single round, specifically;

- Having to hack into atmospherics for gas every round.
- Having to drag two sets of shields up to the surface every round, find somewhere to put them, and build some ugly thing that sticks out into the hall.
- Also finally gave magboots to miners at roundstart, since every other department gets them at roundstart in EVA Storage, despite most rarely actually needing to go EVA.

I also fixed a few things, such as;

- A few ctags in atmospherics that I overlooked in my overhaul of that place.
- The weird incomplete three-door airlock that leads into the engine room.
- Plating being in some weird places in my construction level changes.

Pictures:
https://i.imgur.com/qqax2Im.png
https://i.imgur.com/JazXirv.png
https://i.imgur.com/fetI0sV.png

This makes the gasses for setting up the engine readily available, so that people don't have to spend time breaking into Atmospherics every round when they shouldn't have to, and it ensures people can still do custom setups by giving variety to the gasses available. It also gives a proper cycling airlock into the engine room, which is also easier to carry the canisters through, instead of having to take them one at a time through three consecutive airlocks. It also greatly reduces the time that shields take to be set up by having them already in a dedicated and properly secure room on the surface. They still need to be wrenched into place and have their settings configured properly however, so apprentices can still learn just as easily. It just eliminates the terrible ordeal of carrying them up and building a spot for them every single shift.

### CODE REVIEW

**Comment by Kaedwuff** [comment]
+14,161 −10,059 

Did you run mapmerge?  Or did you change that much stuff?

**Comment by furrycactus** [comment]
I always run mapmerge. Just changed a lot of stuff.

File: maps/_common/areas/station/storage.dm [hunk 1]
```diff
@@ -53,3 +53,8 @@
 	requires_power = 0
 	name = "\improper Test Room"
 	icon_state = "storage"
+
+/area/storage/shields
+	name = "Station Shield Control"
```
Comment (BurgerLUA) [inline maps/_common/areas/station/storage.dm]: This should have \improper at the front of it.

Comment (furrycactus) [reply maps/_common/areas/station/storage.dm]: Got it

**Pull request merged**